### DISCUSSION TITLE
Mirth Connect upgrade, db password unknown

### DISCUSSION THREAD

**Discussion by Coooz** [nextgenhealthcare/connect#5170] [Q&A] [answered]
Hi,

I am just about to install Mirth Connect 3.12, upgrading from 3.5.1. 
I noticed the Server Manager needs to have a username + password for the database:
![image](https://user-images.githubusercontent.com/88899312/166459390-bf4820a7-4974-4124-9343-916d36149ef9.png)
The problem is that this password is more or less lost. Upgrading is no problem if these settings are remembered and still have their current values after the upgrade - but it _is_ a problem if not. And I don't know which one of those two it will be.
Can I upgrade to 3.12 without any worries?

**Comment by mirkoesins** [comment, marked as answer]
Hey, I think you can update without any worries but you can find the combination of username and password in "mirth.properties" in the "conf" folder.

**Reply by Coooz** [reply]
Ah yes, thank you mirkoesins. Indeed, they are mentioned in mirth.properties. 
Quite a relief. 😀
